JS-Algorithms/challenges.js

```diff
@@ -118,7 +168,19 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  let prevFibo = 0;
+  let currentFibo = 1;
+  let nextFibonacci;
+  let index = 0;
+
+  for (; prevFibo.toString().length < n; index++) {
+    nextFibonacci = prevFibo + currentFibo;
```

Comment:
Consider keep consistency in the variable naming, instead of nextFibonacci, maybe use `nextFibo`, given the current code.

Reply:
Done ✅